### ISSUE TITLE
Distinguishing multiple instances of the same applicationName/ReleaseNumber

### ISSUE THREAD

**Issue by openBackhaul** [openBackhaul/ApplicationPattern#681]
As per the current ApplicationPattern, we cannot hold multiple instances of the same application (with same version) in the CONFIGfile.

E.g. for the MediatorInstanceManager this would be required as there are multiple MediatorVMs of the same kind running in parallel.

Currently there is the workaround (distinguishing instances by different applicationNames), but if further cases would occur, straight solutions should be evaluated.

**Comment by openBackhaul** [comment]
Application instances are identified by unique combinations of ApplicationName and ReleaseNumber on the interfaces to many applications, e.g. RO, TAR and ALT.  
Within the applications the CONFIGfile gets searched for these combinations if e.g. TcpServer or OperationNames are to be updated.  
Since an extension to distinguish multiple instances of the same combination of ApplicationName and ReleaseNumber will cause changes at many interfaces and internal processes
- this step must be rated absolutely necessary before
- it cannot be part of a 2.0.n release

A reconsideration was postponed to 2.1.0.
